egs/wsj/s5/local/nnet3/run_lstm.sh

```diff
@@ -24,6 +24,7 @@ non_recurrent_projection_dim=256
 chunk_width=20
 chunk_left_context=40
 chunk_right_context=0
+ivector_interval=10
```

Comment:
I see that you need to specify this value both to the training script and decoding script. So keep this variable specification in the top level script.